rfc6962-bis.xml

```diff
@@ -307,7 +307,7 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
       </t>
       <section title="Certificates">
         <t>
-          Any entity can <xref target="add-chain">submit a certificate</xref> to a log. Since certificates may not be accepted by TLS clients unless logged, it is expected that certificate owners or their CAs will usually submit them.
+          Any entity can <xref target="add-chain">submit a certificate</xref> to a log. It is anticipated that TLS clients will not accept certificates which are not logged, and so it is expected that certificate owners (or their CAs) will submit them.
```

Comment:
Agree with Rob's comments. OTOH, don't see any particular need to change this, not sure what the proposal was supposed to fix.